src/main/java/seedu/address/model/assignment/Schedule.java

```diff
@@ -1,28 +1,26 @@
 package seedu.address.model.assignment;
 
 public class Schedule {
-    private String dummyDeadline = "01-01-2000 0101";
-    private boolean schedule;
-    private Deadline suggestedStartTime;
-    private Deadline suggestedEndTime;
+    private final boolean schedule;
+    private final Deadline suggestedStartTime;
+    private final Deadline suggestedEndTime;
 
     /**
      * Constructor
-     * Returns a false schedule with dummy value
+     * Returns a false schedule
      */
     public Schedule() {
         schedule = false;
-        suggestedStartTime = new Deadline(dummyDeadline);
-        suggestedEndTime = new Deadline(dummyDeadline);
+        suggestedStartTime = null;
+        suggestedEndTime = null;
     }
```

Comment:
If this constructor is used for testing, it may be better to create a ScheduleStub class in the test package for testing :) But this is a minor issue that can be done when you do UnitTesting.

Reply:
Oh, the dummy is for testing, the constructor is not

Reply:
oh okie okie